src/main/scala/scala/xml/quote/quote.scala

```diff
@@ -6,10 +6,7 @@ import scala.xml.quote.internal.QuoteImpl
 package object quote {
   implicit class XmlQuote(ctx: StringContext) {
     object xml {
-      def apply[T](args: T*): scala.xml.Node = macro QuoteImpl.apply[T]
-    }
-    object xmls {
-      def apply[T](args: T*): scala.xml.NodeBuffer = macro QuoteImpl.applySeq[T]
+      def apply[T](args: T*): AnyRef = macro QuoteImpl.apply[T]
```

Comment:
@densh The macro expends to either `scala.xml.NodeBuffer` or a subtype of `scala.xml.Node`.
What should I put as a return type? `AnyRef` will drive IDE crazy...